### ISSUE TITLE
Parse error on code comment

### ISSUE THREAD

**Issue by still-dreaming-1** [dsawardekar/speckle#18]
When I comment out some code inside a spec using /\* */, with strings inside, it has a parse error on the string. Does speckle always fail when you comment out code inside a spec? I know the comments are supposed to work because I can comment out other riml code this way and compile it with riml.

**Comment by dsawardekar** [comment]
I'm not actively working on this at the moment. My riml is quite rusty. Have you tried using # comments instead?

**Comment by still-dreaming-1** [comment]
Yes, actually I was surprised riml itself seemed to support /\* comments and not # comments. I guess I will confirm that part is working properly first. Then if I am still having problems I will come back here and post a simple problem example. Out of curiosity why did you lose interest in riml? I'm asking because the answer could point the way forward for me.
